src/main/java/com/Novi/TechItEasy/helpers/DtoConverters.java

```diff
@@ -0,0 +1,66 @@
+package com.Novi.TechItEasy.helpers;
+
+import com.Novi.TechItEasy.dtos.*;
+import com.Novi.TechItEasy.models.CI_Module;
+import com.Novi.TechItEasy.models.RemoteController;
+import com.Novi.TechItEasy.models.WallBracket;
+
+public class DtoConverters {
+
+    public static void remoteControllerDtoConverter(RemoteController remoteController, RemoteControllerDto dto) {
+
+        dto.id = remoteController.getId();
+        dto.compatibleWith = remoteController.getCompatibleWith();
+        dto.batteryType = remoteController.getBatteryType();
+        dto.name = remoteController.getName();
+        dto.brand = remoteController.getBrand();
+        dto.price = remoteController.getPrice();
+        dto.originalStock = remoteController.getOriginalStock();
+        dto.television = remoteController.getTelevision();
```

Comment:
Je dto bevat nu nog een veld van type Television, maar dat moet een ander type worden. Wanneer je dat verbeterd hebt, zal deze regel (en regels als deze) ook niet meer werken. Je zult dan in deze converter-methode een conversie moeten toepassen.